lib/prefix.c

```diff
@@ -355,6 +355,41 @@ void prefix_copy(union prefixptr udest, union prefixconstptr usrc)
 	}
 }
 
+bool evpn_addr_cmp(const struct evpn_addr *e1, const struct evpn_addr *e2)
```

Comment:
To me it sounds like a `evpn_addr_same()` (naming), not `_cmp`.

Reply:
Hey @ton31337,
Thanks a lot for your review.
I have changed the function's name to `evpn_addr_same`.